.github/workflows/test_gis_examples.yml

```diff
@@ -0,0 +1,60 @@
+name: Test GIS models
+
+on:
+  push:
+    paths:
+      - 'gis/**/*.py'  # If a gis model is modified
+      - 'test_gis_examples.py'  # If the gis test script is modified
+      - '.github/workflows/test_gis_examples.yml'  # If this workflow is modified
+  pull_request:
+    paths:
+      - 'gis/**/*.py'
+      - 'test_gis_examples.py'
+      - '.github/workflows/test_gis_examples.yml'
+  workflow_dispatch:
+  schedule:
+    - cron: '0 6 * * 1'  # Monday at 6:00 UTC
+
+jobs:
+  # build-stable:
```

Comment:
When will this be enabled?

Reply:
Good question. I think we will want to make a backport to the [`mesa-2.x`](https://github.com/projectmesa/mesa-examples/tree/mesa-2.x) branch with examples, since Mesa-Geo is meant to work with Mesa 2.3.

So then we will make te split of testing pre-releases + git on the main branch, and stable on the Mesa-Geo 0.8.x branch.